[Android] Add more prebuilt artifacts

kirklandsign opened the pull request:
Build for different ABI in prebuild.

Next step: Build and use libllama_runner.so instead of static libs, so that we can combine RE2 and ABSL stuff together.

huydhn commented:
Just FYI, the authentication part with AWS doesn't work with forked PR, i.e. `kirklandsign:android-prebuilt` from your fork, so it's failing expectedly https://github.com/pytorch/executorch/actions/runs/8792470295/job/24128672297?pr=3215.  I guess we can ignore it in this case.

kirklandsign committed:
Add more prebuilt artifacts (#3215)

Summary:
Build for different ABI in prebuild.

Next step: Build and use libllama_runner.so instead of static libs, so that we can combine RE2 and ABSL stuff together.


Reviewed By: huydhn

Differential Revision: D56438244

Pulled By: kirklandsign
